tests/functional/bundle.yaml

```diff
@@ -0,0 +1,140 @@
+description: A minimal two-machine Kubernetes cluster, appropriate for development.
+series: focal
+machines:
+  '0':
+    constraints: cores=2 mem=4G root-disk=16G
+    series: focal
+  '1':
+    constraints: cores=2 mem=4G root-disk=16G
+    series: focal
+  '2':
+    constraints: cores=4 mem=4G root-disk=16G
+    series: focal
+  '3':
+    constraints: cores=2
+    series: bionic
+  '4':
+    constraints: cores=2
+    series: bionic
+  '5':
+    constraints: cores=2
+    series: bionic
+applications:
+  containerd:
+    charm: cs:~containers/containerd-160
```

Comment:
I was wondering if `cs:~containers/containerd` should be used instead of the same with `-160`.

Reply:
We discussed to keep it aligned with the latest release. In the future, maintainers should bump the revnos to the latest release.